win32ss/user/ntuser/nonclient.c

```diff
@@ -285,6 +300,7 @@ DefWndDoSizeMove(PWND pwnd, WORD wParam)
    }
    else  /* SC_SIZE */
    {
+      RECTL_vSetEmptyRect(&pwnd->InternalPos.NormalRect); /* CORE-19160 */
```

Comment:
We do already have a PR for that distinct added line from Doug Lyons. I do suggest to let him merge his PR first, and then take that change out of yours. His fix is very distinct, while your PR touches lots of things additionally. This will also require you to mention less CORE-IDs in your PR.

Reply:
@whindsaks I do have to unresolve this conversation again, sry. When I mentioned this earlier, and encouraged that line to be excluded from your PR, I still expected DougLyons to commit his https://github.com/reactos/reactos/pull/5685 on his own beforehand. But meanwhile Doug closed his PR, so it might be worth reconsidering pushing that line as well! Sry for the confusion, that was due to the nature of parallel PRs.